testing-guidance/workflow-tests/multiplayer/multi-machine-multiplayer-workflow-tests.md

```diff
@@ -0,0 +1,78 @@
+**Testing Multiplayer Launchers on Multiple Machines**
+======================================================
+
+This test is intended to verify successful connection between separate machines on the same network.  Ideally this is completed with a Linux machine as server and any number of windows machines as clients, but it can be completed with any combination.
+
+**Prerequisites**
+=================
+
+**All machines used for this test should be on the same Network**
+
+**Acquire and Compile MultiplayerSample project** ( [https://github.com/o3de/o3de-multiplayersample](https://github.com/o3de/o3de-multiplayersample) )
+
+**Set Up for the Server Machine:**
+
+These steps only need to be done on the 1 machine that will be acting as the server/host for the test
+
+Windows:
```

Comment:
Instructions are for WIndows Server, yet you call out common case should be Linux servers. Maybe flip Linux first?